sktime/data/util.py

```diff
@@ -84,14 +93,17 @@ def timeshifted_split(inputs, lagtime: int, chunksize: int = 1000, stride: int =
             data = data[:-lagtime][::stride]
         else:
             data = data[0::stride]  # otherwise data is empty as slice over `data[:-0]`
+        ix = np.arange(len(data))  # iota range over data
```

Comment:
iota?

Reply:
it is sometimes used to denote integer ranges, this stackoverflow answer explains it better than i could do it here: https://stackoverflow.com/a/9244949/2871028

Reply:
today i learned!